examples/server/README.md

```diff
@@ -446,7 +446,7 @@ These words will not be included in the completion, so make sure to add them to
 
 **Response format**
 
-- Note: When using streaming mode (`stream`), only `content` and `stop` will be returned until end of completion.
+- Note: In streaming mode (`stream`), only `content` and `stop` will be returned until end of completion. Responses currently use a format resembling the [Server-sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events) standard, but the `EventSource` interface cannot be used due to its lack of `POST` request support.
```

Comment:
```suggestion
- Note: In streaming mode (`stream`), only `content` and `stop` will be returned until end of completion. Responses are sent using the [Server-sent events](https://html.spec.whatwg.org/multipage/server-sent-events.html) standard. Note: the browser's `EventSource` interface cannot be used due to its lack of `POST` request support.
```

Removing the word "resembling" because the SSE standard doesn't specify if it supports GET or POST request. So we are already compliant with the standard. It's the browser that only implement GET for `EventSource`